lib/node_modules/@stdlib/stats/base/dcumax/README.md

```diff
@@ -166,6 +165,120 @@ console.log( y );
 
 <!-- /.examples -->
 
+<!-- C interface documentation. -->
+
+* * *
+
+<section class="c">
+
+## C APIs
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- C usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```c
+#include "stdlib/stats/base/dcumax.h"
+```
+
+#### stdlib_strided_dcumax( N, \*X, strideX, \*Y, strideY )
+
+Computes the cumulative maximum of double-precision floating-point strided array elements.
+
```

Comment:
Missing a C example.

Reply:
What should i write here for a void function? had some confusions about that so i left it for now @kgryte

Reply:
See, e.g., https://github.com/stdlib-js/stdlib/tree/develop/lib/node_modules/%40stdlib/blas/base/dcopy.

Reply:
Also, rather than omit entirely, better to insert a TODO, so that reviewers know that something is missing. Otherwise, can be easy to miss.